tests/unit_tests/corsika/test_corsika_default_config.py

```diff
@@ -9,27 +9,26 @@ def compare_lists_ignoring_units(list1, list2):
 
 
 def test_primary_setter():
-    config = CorsikaDefaultConfig()
-    config.primary = "gamma"
+    config = CorsikaDefaultConfig(primary="gamma", zenith_angle=20.0 * u.deg)
     assert config.primary == "gamma"
 
 
 def test_primary_setter_invalid(caplog):
-    config = CorsikaDefaultConfig()
+    config = CorsikaDefaultConfig(primary="gamma", zenith_angle=20.0 * u.deg)
     with pytest.raises(ValueError):
         config.primary = "invalid_particle"
```

Comment:
Good test. Could we also test setting from the beggining an invalid particle? E.g.,
 `config = CorsikaDefaultConfig(primary="invalid_particle", zenith_angle=20.0 * u.deg)`

Reply:
That's a good idea. I modified a few tests so that I check also the init itself separately. You can take another look if you wish.

Reply:
I am fine with the new changes, though some (seemly?) unrelated tests failed